target/linux/mediatek/dts/mt7981b-iptime-ax3000m.dts

```diff
@@ -0,0 +1,276 @@
+// SPDX-License-Identifier: (GPL-2.0 OR MIT)
+
+/dts-v1/;
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include "mt7981.dtsi"
+
+/ {
+	model = "ipTIME AX3000M";
+	compatible = "iptime,ax3000m", "mediatek,mt7981-spim-snand-rfb";
+
+	aliases {
+		label-mac-device = &gmac0;
+		led-boot = &led_cpu;
+		led-failsafe = &led_cpu;
+		led-running = &led_cpu;
+		led-upgrade = &led_cpu;
+		serial0 = &uart0;
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n8";
+		bootargs-append = " ubi.mtd=ubi rootfstype=squashfs";
+		bootargs-replace = "boot_from=B|ubi.block=0,rootfs2 root=/dev/ubiblock0_4";
```

Comment:
Is this only needed when it boots from B and not when it boots from A?

Reply:
yes, this is only needed when boot_from is B.
when boot_from is A, `root=` is not given and rootfs is automatically setted by this [patch](https://github.com/openwrt/openwrt/blob/main/target/linux/generic/pending-6.6/493-ubi-set-ROOT_DEV-to-ubiblock-rootfs-if-unset.patch).